server/lib/http-error.js

```diff
@@ -21,7 +21,8 @@ class HttpError {
     const userFriendlyError = {
       statusMessage,
       status: statusCode,
-      message: error.userMessage || error
+      message: error.userMessage || error,
+      localeInfo: localeInfo
```

Comment:
You can also just do `localInfo` like line 22 if you want, but this is fine too.